### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by remstone7**
create a new version 2.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 [Unreleased]: https://github.com/klaviyo/magento2-klaviyo/compare/1.2.4...HEAD
```
Comment (njparadis) [inline CHANGELOG.md:41]: Do we need to update this URL as well?

**Review by smoucka — COMMENTED** [review]
Should this just be a minor change? Are we breakin anything?

File: CHANGELOG.md (hunk 1 above)
Comment (remstone7) [reply CHANGELOG.md:41]: yes we do, totally missed this one.. thanks!

**Comment by remstone7** [comment]
> Should this just be a minor change? Are we breakin anything?

adding webhook functionality, app only supports version 2.0.0 for webhooks...

**Review by smoucka — APPROVED** [review]

**Review by njparadis — APPROVED** [review]

**Pull request merged**